src/lib/file-util.mjs

```diff
@@ -22,15 +20,8 @@ export function fixPath(path) {
  * @returns {boolean}
  */
 export function dirExists(path) {
-  const stat = fs.statSync(fixPath(path), {
-    // returns undefined instead of throwing if the file doesn't exist
-    throwIfNoEntry: false,
-  });
-  if (stat === undefined || !stat.isDirectory()) {
-    return false;
-  } else {
-    return true;
-  }
+  const fs = container.resolve("fs");
+  return fs.existsSync(fixPath(path));
```

Comment:
this doesn't assert that it's a directory - just that the path exists. rename or change?